sci-astronomy/siril/siril-1.0.0.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson xdg-utils
```

Comment:
Inheriting `xdg.eclass` instead of `xdg-utils.eclass` would make the ebuild a bit shorter. This eclass exports the *_update functions to the relevant phases automatically for you.

Reply:
Thanks for the suggestion. I fixed it.